pkgs/tools/security/ibm-sw-tpm2/default.nix

```diff
@@ -30,12 +35,14 @@ stdenv.mkDerivation rec {
 
   sourceRoot = "src";
 
+  inherit makefile;
+
```

Comment:
```suggestion
```

Reply:
Are you suggesting to remove the inherit statement? I am still a newcomer to Nix, but I have a hard time understanding how this should work...

Reply:
Yes.

Reply:
I don't get it either @wegank. How would that even work? Can you provide more feedback than just one word when reviewing code and making such suggestions?

https://nixos.org/manual/nixpkgs/stable/#var-stdenv-makefile

Reply:
Oh, I'm completely wrong, sorry.